filebeat/tests/system/test_registrar.py

```diff
@@ -37,6 +40,9 @@ def test_registrar_file_content(self):
                 "Processing 5 events"),
             max_timeout=15)
 
+        # Wait 2s to make sure the state is written once again
+        time.sleep(2)
```

Comment:
This is / was a little bit more complex as I had to make sure state was written again. But with the most recent change to only write state on changes, this is not required anymore. I improved it now by checking for the number of states written before adding line and having it occur more afterwards.